app/services/entry_service.py

```
@@ -95,6 +96,349 @@ def _refresh_entry_date(self, entry: Entry) -> None:
+                log_info("Media deleted", media_id=str(media.id), user_id=str(user_id))
+                return True, None, None
+            else:
+                # Return deletion info for caller to handle post-commit
+                log_debug(
+                    "Media DB record deleted (files pending post-commit)",
+                    media_id=str(media.id),
+                    user_id=str(user_id)
+                )
+                return True, file_deletion_info, asset_id_to_remove
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)